index.ts

```diff
@@ -302,10 +308,8 @@ class OptionsSync<UserOptions extends Options> {
 		return serialize(form, {include});
 	}
 
-	private _handleStorageChangeOnForm(changes: Record<string, any>, areaName: string): void {
-		if (
-			areaName === 'sync'
```

Comment:
This should also exist, to ensure we're looking at the expected key in the expected storage